internal/module/resource.go

```diff
@@ -0,0 +1,13 @@
+package module
+
+import (
+	"github.com/terraform-docs/terraform-docs/pkg/tfconf"
+)
+
+type resourcesSortedByType []*tfconf.Resource
```

Comment:
Would you please create `resource_test.go` similar to `input_test` and `output_test` and ...?